src/api.tsx

```diff
@@ -250,6 +250,41 @@ export async function addMedia(name: string, type: "icon" | "photo", file: File)
 
 }
 
+export async function addRoomInfo(name: string, pic: File): Promise<GuideInfo> {
```

Comment:
I think this is now the third function that uploads a file, shall we refactor to one generic that all 3 are calling it?